src/NLog/Layouts/LayoutParser.cs

```diff
@@ -431,9 +431,9 @@ private static LayoutRenderer GetLayoutRenderer(ConfigurationItemFactory configu
             }
             catch (Exception ex)
             {
-                if (LogManager.ThrowConfigExceptions ?? LogManager.ThrowExceptions)
+                if (throwConfigExceptions ?? LogManager.ThrowConfigExceptions ?? LogManager.ThrowExceptions)
                 {
-                    throw;
+                    throw;  // TODO throw NLogConfigurationException
```

Comment:
Is this for NLog 5? Please make it `TODO NLog 5` then :)

Reply:
Fixed